packages/server/postgres/queries/generated/getTeamPromptResponsesByIdsQuery.ts

```diff
@@ -26,13 +27,13 @@ export interface IGetTeamPromptResponsesByIdsQueryQuery {
   result: IGetTeamPromptResponsesByIdsQueryResult;
 }
 
-const getTeamPromptResponsesByIdsQueryIR: any = {"name":"getTeamPromptResponsesByIdsQuery","params":[{"name":"ids","codeRefs":{"defined":{"a":53,"b":55,"line":3,"col":9},"used":[{"a":213,"b":215,"line":6,"col":13}]},"transform":{"type":"array_spread"}}],"usedParamSet":{"ids":true},"statement":{"body":"SELECT \"id\", \"createdAt\", \"updatedAt\", \"meetingId\", \"userId\", \"sortOrder\", \"content\", \"plaintextContent\" FROM \"TeamPromptResponse\"\nWHERE id in :ids","loc":{"a":69,"b":215,"line":5,"col":0}}};
+const getTeamPromptResponsesByIdsQueryIR: any = {"name":"getTeamPromptResponsesByIdsQuery","params":[{"name":"ids","codeRefs":{"defined":{"a":53,"b":55,"line":3,"col":9},"used":[{"a":318,"b":320,"line":6,"col":46}]},"transform":{"type":"array_spread"}}],"usedParamSet":{"ids":true},"statement":{"body":"SELECT CONCAT('teamPromptResponse:', \"id\") as id, \"createdAt\", \"updatedAt\", \"meetingId\", \"userId\", \"sortOrder\", \"content\", \"plaintextContent\", to_json(\"reactjis\") as \"reactjis\" FROM \"TeamPromptResponse\"\nWHERE CONCAT('teamPromptResponse:', \"id\") in :ids","loc":{"a":69,"b":320,"line":5,"col":0}}};
 
 /**
  * Query generated from SQL:
  * ```
- * SELECT "id", "createdAt", "updatedAt", "meetingId", "userId", "sortOrder", "content", "plaintextContent" FROM "TeamPromptResponse"
- * WHERE id in :ids
+ * SELECT CONCAT('teamPromptResponse:', "id") as id, "createdAt", "updatedAt", "meetingId", "userId", "sortOrder", "content", "plaintextContent", to_json("reactjis") as "reactjis" FROM "TeamPromptResponse"
+ * WHERE CONCAT('teamPromptResponse:', "id") in :ids
```

Comment:
-1 i'd argue this is a leaky abstraction, the database shouldn't need to be aware that we prefix its IDs in order to make them globally unique for the client. Ideally, we should be able to change the prefix that gets sent to the client without affecting anything that touches the database.

The standard practice that I've seen (https://github.com/graphql/graphql-relay-js#object-identification) does the following:
- when a GUID comes in via a mutation argument, it's transformed to a database ID at the top of the resolve function
- all server functions (db helpers, dataloaders, lookups, etc) use the database ID
- in the GraphQL type resolve function, the database ID is transformed into a GUID before going back to the client

In doing so, the scope of the prefix is limited to just the ingress/egress.
happy to debate or change thinking though! just want to make sure we do this the same way with every entity (e.g. https://github.com/ParabolInc/parabol/blob/c431a3b26753a7cb57e8820677df43c0a766a9e5/packages/server/graphql/types/Poll.ts#L26)

Reply:
In the interest of getting this landed (especially since it's still behind a feature flag), I've reverted the change that makes this a leaky abstraction, back towards the existing GQLID patterns.

I'm going to put some thought into how this should be refactored in the near-/medium-future and start a discussion with product.